### ISSUE TITLE
Multiple single exe apps not working under IIS on the same machine

### ISSUE THREAD

**Issue by RomanHubyak** [dotnet/core#4462]
I have published a single exe app to IIS and it works as expected.

When I publish another single exe app to the same IIS on the same machine it doesn't work at all. Here are errors:

![image](https://user-images.githubusercontent.com/46841287/77617153-150b3b80-6f3c-11ea-847a-5608a27116a3.png)

![image](https://user-images.githubusercontent.com/46841287/77617187-29e7cf00-6f3c-11ea-822b-1dda8713bf0c.png)

I checked the directory [C:\WINDOWS\TEMP\.net\] and it already exists and contains unpacked first app.

**Comment by igoventura** [comment]
The problem is that the application is trying to bind to port 22110. Check if another application is using this port already or if the port is logically created on IIS.

**Comment by romanryasne2** [comment]
I don't think so. Deleting TEMP folder made second app to work but the first app crashed with the same issue.

**Comment by igoventura** [comment]
What is the port used by the 2 apps? The second one is using 22110. And the first one?

**Comment by RomanHubyak** [comment]
It turned out that .net directory is created without any permissions for other users/apps. That's why it fails.

**Comment by swaroop-sridhar** [comment]
@RomanHubyak  Which version of Windows is this? and which version of .net core, asp.net? 
Do you have the default configuration for %temp%, or has anything changed? 

@DamianEdwards .net core 3.x single-file apps extract to %temp%\.net\... assuming that %temp% is a user-private temp directory. Does IIS use the system-temp directory instead? Thanks.
